### PR TITLE
Update links in Additional Resources section

### PR DESCRIPTION
**Pull request by mmattar**
### Proposed change(s)

Got rid of old links and added a link to Hummingsbird course

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [X] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,11 @@ https://github.com/Unity-Technologies/ml-agents.
 
 ## Additional Resources
 
-We have published a series of blog posts that are relevant for ML-Agents:
+We have a Unity Learn course,
+[ML-Agents: Hummingsbird](https://learn.unity.com/course/ml-agents-hummingbirds),
+that provides a gentle introduction to Unity and the ML-Agents Toolkit.
```
Comment (unityjeffrey) [inline README.md:98]: gentle?

**Review by unityjeffrey — APPROVED** [review]

**Comment by chriselion** [comment]
Should we link to the CodeMonkey videos too?

File: README.md (hunk 1 above)
Comment (mmattar) [reply README.md:98]: Intended to highlight that it's an easy to follow tutorial.

**Comment by mmattar** [comment]
> Should we link to the CodeMonkey videos too?

That makes sense to me. @unityjeffrey - any reason not to?

@chriselion - can you slack me the links?

**Comment by mmattar** [comment]
> > Should we link to the CodeMonkey videos too?
> 
> That makes sense to me. @unityjeffrey - any reason not to?
> 
> @chriselion - can you slack me the links?

Added

File: README.md [hunk 2]
```diff
@@ -93,7 +93,16 @@ https://github.com/Unity-Technologies/ml-agents.
 
 ## Additional Resources
 
-We have published a series of blog posts that are relevant for ML-Agents:
+We have a Unity Learn course,
```
Comment (chriselion) [inline README.md:96]:
```suggestion
We have a Unity Learn course 
```
grammar nit

**Review by chriselion — APPROVED** [review]

**Pull request merged**